src/util/server.c

```diff
@@ -451,7 +452,8 @@ static const char *get_pid_path(void)
 #endif
 }
 
-int server_setup(const char *name, int flags,
+int server_setup(const char *name, bool is_responder,
```

Comment:
It might be better to use 'const char *chain_id_fmt` instead of boolean to make it more generic. Otherwise it looks good to me, thank you.

Reply:
I thought `is_responder` could be re-usable for some other need later on inside `server_setup()`, rather than only for the chain_id_fmt use case.

Reply:
Ok, I'm not going to insist on this.